.github/workflows/test.yaml

```diff
@@ -19,10 +19,10 @@ jobs:
     - name: Install libcurl
       run: sudo apt-get update && sudo apt-get install libssl-dev libcurl4-openssl-dev
     - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
-    - name: Set up Python 3.9
+    - name: Set up Python 3.10
       uses: actions/setup-python@13ae5bb136fac2878aff31522b9efb785519f984 # v4.3.0
       with:
-        python-version: 3.9
+        python-version: '3.10'
```

Comment:
If you don't quote this, will it load python 3.1? 😅

Reply:
It caused an error, I forget the details - I **think** it was just looking at the first 3 characters and trying to use 3.1. 😞